src/software/backend/handheld_controller_backend.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "software/backend/backend.h"
+#include "software/backend/output/radio/radio_output.h"
+#include "software/parameter/dynamic_parameters.h"
+
+class HandheldControllerBackend : public Backend
+{
+   public:
+    static const std::string name;
+
+    HandheldControllerBackend();
+
+    /**
+     * Creates a new PrimitiveGenerator
+     *
+     * @param controller_input_config The config for the HandheldControllerBackend
+     */
+    HandheldControllerBackend(
+        std::shared_ptr<const HandheldControllerInputConfig> controller_input_config);
+
+   private:
+    static constexpr int DEFAULT_RADIO_CONFIG = 0;
+
+    void onValueReceived(ConstPrimitiveVectorPtr primitives) override;
+
+    // The interface that lets us send primitives to the robots over radio
+    RadioOutput radio_output;
```

Comment:
Just so we're totally clear, this will not work with Frankie?

Reply:
Currently that's correct. We would need a new wifi output backend to work with Frankie.